src/stripe.test.jsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-addons-test-utils';
+import { findComponentsWithType } from 'meetup-web-mocks/lib/testUtils';
+
+import Stripe, {
+	STRIPE_CLASS,
+	STRIPE_COLLECTION_CLASS,
+	STRIPE_INVERTED_CLASS,
+	STRIPE_HERO_CLASS,
+} from './Stripe';
+
+describe('Stripe', function() {
+
+	it('exists', function() {
+		const stripe = TestUtils.renderIntoDocument(<Stripe />);
+		const stripeNode = ReactDOM.findDOMNode(stripe);
+
+		expect(stripeNode).not.toBeNull();
+	});
+
+	it(`check that default component has '${STRIPE_CLASS}' class`, function() {
+		const stripe = TestUtils.renderIntoDocument(<Stripe />);
+		const stripeNode = ReactDOM.findDOMNode(stripe);
+
+		expect(stripeNode.classList).toContain(STRIPE_CLASS);
+	});
+
+	describe('collection', () => {
+		let stripeCollection;
+		beforeEach(() => {
+			stripeCollection = TestUtils.renderIntoDocument(<Stripe collection/>);
+		});
+		afterEach(() => {
+			stripeCollection = null;
+		});
+		it(`check that default component has '${STRIPE_COLLECTION_CLASS}' class`, function() {
+			const stripeNode = ReactDOM.findDOMNode(stripeCollection);
+
+			expect(stripeNode.classList).toContain(STRIPE_COLLECTION_CLASS);
```

Comment:
just thinking about ways this could break, and wondering if it's worth throwing in a quick check that `STRIPE_CLASS` is also applied whenever a variant is specified?

```js
expect(stripeNode.classList).toContain(STRIPE_CLASS);
```